operator/buffer/buffer.go

```diff
@@ -3,12 +3,16 @@ package buffer
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 
 	"github.com/open-telemetry/opentelemetry-log-collection/entry"
 	"github.com/open-telemetry/opentelemetry-log-collection/operator"
 )
 
+// ErrBufferedClosed is an error to indicate an operation was attempt on a buffer after it was closed
+var ErrBufferedClosed = errors.New("buffer is closed")
```

Comment:
```suggestion
// ErrBufferClosed is an error to indicate an operation was attempt on a buffer after it was closed
var ErrBufferClosed = errors.New("buffer is closed")
```

Reply:
I'll change this in a fixup as it's used elsewhere.